### ISSUE TITLE
void functions

### ISSUE THREAD

**Issue by JoePelz** [riolet/ritchie#17]
void functions that return nothing. Are they a part of the language?
Assuming functions can be non-pure (i.e. have side effects) void functions have a place and should be possible to write.

possibilities:
```
:myPrint Integer i
  print i

Void: myPrint Integer i
  print i

Null: myPrint Integer i
  print i
```
Presumably the return statement isn't necessary because blocks are marked by indentation instead of braces.

**Comment by JoePelz** [comment]
Resolved. Decided upon:
```
:myPrint Integer i
  print i
```

**Comment by JoePelz** [comment]
Reopening. This needs the actual fix to allow void functions before closing it.

**Comment by JoePelz** [comment]
Closing. Void functions work now!
Fixed in 35eeff36aaa309a7573c882eaa836d91f6963abe and 253495e95b45db013b7a48d24bc84e73be012de0
